### DISCUSSION TITLE
How to deploying to render

### DISCUSSION THREAD

**Discussion by brandscart** [serverpod/serverpod#2463] [Q&A] [answered]
Hi,

Iam first time in hosting of a server. so please let me know how to deploy my serverpod project to render. I believe choosing render is a better option when looking the price it has free tier. i can't know  the document given in the serverpod well.

So if you can, give me some reference document too.
thanks

**Comment by vlidholt** [comment, marked as answer]
Hi @brandscart,

From a quick look, Hostinger seem to be better suited for hosting Wordpress websites. For hosting your own server, you will need an instance where you can run custom code, in this case Dart.

I may be incorrect, so if someone has gotten Serverpod to run on Hostinger, please post below! :)

**Reply by iampopal** [reply]
I need to deploy to fly.io, any suggestions?
